src/fontMetrics.js

```diff
@@ -106,7 +106,10 @@ var metrics = {
             return sigma21ScriptScript;
         }
         throw new Error("Unexpected style size: " + style.size);
-    }
+    },
+
+    // \nulldelimiterspace=1.2pt, pg. 344 TeXbook
+    nulldelimiterspace: 1.2 / ptPerEm
```

Comment:
For all pt-based metrics, we assume 10 pt per em. This means we don't need to know the font size in order to render the expressions, like in server mode when we have absolutely no way of knowing. We might want to bump this up a little to 12 or something but there's only a few place where it matters and it makes only a minor difference.